include/psa/crypto_extra.h

```diff
@@ -572,8 +572,8 @@ psa_status_t psa_get_key_domain_parameters(
 /** \defgroup psa_tls_helpers TLS helper functions
  * @{
  */
-
-#if defined(MBEDTLS_ECP_LIGHT)
+#include <mbedtls/pk.h>
```

Comment:
I have to think a bit about this. Yes, I can move `MBEDTLS_PK_HAVE_ECC_KEYS` to `build_info.h` and I guess it should work, but at the same time I should also move `MBEDTLS_PK_USE_PSA_EC_DATA` because the former depends on the latter. I like the way these symbols are defined because it's easy for someone who tries to understand the code that the PK module can have EC support in one way or the other (legacy vs new).
At the same time I would like to not redefine the same thing in different places (one in `pk.h` for PK usage and one in `build_info` or anywhere else for PSA usage). This usually leads to confusion and easy misalignment in the future.

I'll see what I can do here ;)